### ISSUE TITLE
Ways to contribute

### ISSUE THREAD

**Issue by parruc** [LibrePhotos/librephotos#158]
I'd like to contribute to this project in my spare time cause I'm using it and I find it AWESOME.

I happen to develop in Django Is there a way I can contribute to this project?

I could not find a donate button on the repository description, if there is not one, it could be a good idea  to add it to support yout huge efforts. I would contribute for example ;-)

**Comment by derneuere** [comment]
Hey @parruc 

do you have experience in finding [memory leaks](https://github.com/LibrePhotos/librephotos/issues/93) or how to implement rq workers with django? I have trouble getting scan_photos to run as a rq job https://github.com/LibrePhotos/librephotos/blob/92393367028ce10d92f68617547adab517c1fc5d/api/views.py#L1573-L1582
You can also pick other backend related issues.

I will look into adding a donating button :)

**Comment by parruc** [comment]
I usually use uwsgi spoolers for long running jobs but I guess the concepts are quite similar. I tried to setup a development environment to look into it this weekend but I was not able to. Is there a development bootstrap guide?

I found quite strange that there is one unique repository containing both docker setup and librephotos code mixed together, and that code folder is copied during image build not allowing devs to mount it.

Would not be a good idea to split the two repos so that docker mounts the code folder as a volume allowing developers to edit actual application code directly from his/her host?
